h/static/scripts/permissions.coffee

```diff
@@ -63,6 +72,29 @@ module.exports = ['session', (session) ->
       admin: [session.state.userid]
     }
 
+  private: private_
```

Comment:
This is mainly CoffeeScript weirdness, but I find this a bit harder to read now that we switch apparently without warning from defining functions to an (implicit) returned object.

I think it would be less confusing if you stuck with the old way of just defining the object inline, and using `this.private` and `this.shared` in `default`.